weave/legacy/weave/refs_internal.py

```diff
@@ -0,0 +1,215 @@
+# This file contains the definition for "Internal" weave refs. These should never be
```

Comment:
This is not legacy!

Reply:
I think there is some dependency on the refs code in query service.  There is a duplicate file at `trace/refs_internal.py` for the trace code.